src/billing/models.py

```diff
@@ -2002,12 +2020,12 @@ def capture(self):
         # based on the stripe charge tied to the invoice
         charge = PaymentCharge.objects.create(
             payment_method=payment_method,
-            price=stripe_charge["amount"] / 100,
-            description=stripe_charge["description"],
+            price=stripe_payment_intent["amount"] / 100,
```

Comment:
@vegu this should probably be a decimal instead of float?

Reply:
its a decimal field in the db schema, and django will convert it